tests/e2e/specs/audit.js

```diff
@@ -0,0 +1,243 @@
+    cy.route('POST', Cypress.env('VUE_APP_FIREROAD_URL') + '/requirements/progress/girs/', girs);
+    cy.route('POST', Cypress.env('VUE_APP_FIREROAD_URL') + '/requirements/progress/major21M-1/', major21M1);
+    cy.route('POST', Cypress.env('VUE_APP_FIREROAD_URL') + '/requirements/progress/major16/', major16);
+    cy.route(Cypress.env('VUE_APP_FIREROAD_URL') + '/requirements/list_reqs/',
+      objectSlice(reqs, ['girs', 'major21M-1', 'major16']));
+
+    cy.visit('/');
+
+    // Check GIRs already present in audit
+    cy.getByDataCy('auditItemgirs')
+      .should('exist')
+      .should('contain', 'GIRs');
+
+    // Search and enter 16 major
+    cy.getByDataCy('auditMajorChips')
+      .type('16{enter}{backspace}{backspace}{esc}');
+
+    // Check that Course 16 is in audit now
+    cy.getByDataCy('auditItemmajor16')
+      .should('exist')
+      .should('contain', 'Aerospace Engineering');
+
+    // Pull up course selection menu again
+    cy.getByDataCy('auditMajorChips')
+      .click();
+
+    // Click on major 21M-1
+    cy.get('.v-menu__content.menuable__content__active a.v-list__tile')
+      .contains('21M-1')
+      .click();
+
+    // Close menu
+    cy.getByDataCy('auditMajorChips')
+      .type('{esc}');
+
+    // Check that Course 21M-1 is in audit now
+    cy.getByDataCy('auditItemmajor21M-1')
+      .should('exist')
+      .should('contain', 'Music');
+
+    // Press close button on 16 chip
+    cy.getByDataCy('auditMajorChips')
+      .parents('.v-select--chips')
+      .within(() => {
+        cy.get('.v-chip')
+          .contains('16')
+          .within(() => {
+            cy.get('.v-chip__close')
+              .click();
+          });
+      });
+
+    // Check that Course 16 is not in audit
+    cy.getByDataCy('auditItemmajor16')
+      .should('not.exist');
+
+    // But the other ones still are
+    cy.getByDataCy('auditItemgirs')
+      .should('exist');
+    cy.getByDataCy('auditItemmajor21M-1')
+      .should('exist');
+
+    // Tab over to GIRs and delete it
+    cy.getByDataCy('auditMajorChips')
+      .click()
+      .type('{leftarrow}{leftarrow}{backspace}')
+      .type('{esc}');
+
+    // Check GIRs not in audit
+    cy.getByDataCy('auditItemgirs')
+      .should('not.exist');
+
+    // But others are unchanged
+    cy.getByDataCy('auditItemmajor16')
+      .should('not.exist');
+    cy.getByDataCy('auditItemmajor21M-1')
+      .should('exist');
+
+    // Pull up course selection menu again
+    cy.getByDataCy('auditMajorChips')
+      .click();
+
+    // Click on major 21M-1 to deselect it
+    cy.get('.v-menu__content.menuable__content__active a.v-list__tile')
+      .contains('21M-1')
+      .click();
+
+    // Pull up course selection menu again
+    cy.getByDataCy('auditMajorChips')
+      .type('{esc}');
+
+    // Check that Course 21M-1 is not in audit now
+    cy.getByDataCy('auditItemmajor21M-1')
+      .should('not.exist');
+
```

Comment:
Can you add a test for removing a major using the info menu -> remove requirement?

Reply:
Added this to the add + remove majors test